demucs/api.py

```diff
@@ -264,6 +264,26 @@ def separate_tensor(
         """
         if sr is not None and sr != self.samplerate:
             wav = convert_audio(wav, sr, self._samplerate, self._audio_channels)
+        if wav.max() == wav.min():
+            try:
+                self._callback(  # type: ignore
+                    _replace_dict(
+                        self._callback_arg,
+                        ("state", "end"),
+                        ("model_idx_in_bag", (len(self._model.models) - 1)
+                         if isinstance(self._model, BagOfModels) else 0),
+                        ("shift_idx", self._shifts - 1),
+                        ("segment_offset", wav.shape[1]),
+                        ("audio_length", wav.shape[1]),
+                        ("models", len(self._model.models) if isinstance(self._model, BagOfModels)
+                         else 1)
+                    )
+                )
+            except KeyboardInterrupt:
+                raise
+            except Exception:
```

Comment:
no silent catching of exception, this should be removed. Then both except :  block should be removed.